rho/ansible_utils.py

```diff
@@ -69,16 +69,20 @@ def log_yaml_inventory(label, inventory):
 
     :param inventory: A dictionary of the ansible inventory
     """
-    alpha = inventory.get('alpha')
-    hosts_dict = alpha.get('hosts')
-    vars_dict = alpha.get('vars')
+
     redact_key_list = ['ansible_become_pass', 'ansible_ssh_pass']
 
-    # pylint: disable=unused-variable
-    for host, host_dict in iteritems(hosts_dict):
-        host_dict = redact_dict(redact_key_list, host_dict)
+    groups = list(inventory.keys())
+
+    for group in groups:
+        hosts_dict = inventory[group].get('hosts')
+        vars_dict = inventory[group].get('vars')
+
+        # pylint: disable=unused-variable
+        for host, host_dict in iteritems(hosts_dict):
```

Comment:
Rewriting this as `for host_dict in hosts_dict.values()` should avoid the pylint disable and won't need the host variable.